python/speech-to-text/README.MD

```diff
@@ -0,0 +1,47 @@
+
+# Speech To Text using Python (google, aws, azure)
+
+### Example Input
+```
+{"provider":"aws","language":"en","audio":"iVBORw0KGgoAAAANSUhEUgAAAaQAAALiCAY...QoH9hbkTPQAAAABJRU5ErkJggg=="}
+```
+
+key
+a dict which provides `key` and `region` of the asked cloud provider in payload like.
+key = 
+```
+{        
+    "GOOGLE_APPLICATION_CREDENTIALS":r"E:\Projects\speech-to-text\speech-364611-0c4743e21df2.json",
+    "aws_access_key_id": "AKIAVJDDVDNFSAPCJK",    
+    "aws_secret_access_key": "xXAmNVe000HiSOdf22dNi2qw122YD",
+    "azure_speech_key": "6bcz1347581ce4c1485df491bga0e72e",
+    "region":"centralIndia"
+}
```

Comment:
These are Function variables. They should be in req.variables and documented as such. For a sample of what this README should look like, refer to https://github.com/open-runtimes/examples/blob/fa253884ade36029762efff90bcb206d7cc984bc/python/deepgram-transcribe-video/README.md